src/quay_push.sh

```diff
@@ -26,3 +27,5 @@ docker --config="$DOCKER_CONF" login -u="$QUAY_USER" -p="$QUAY_TOKEN" quay.io
 docker --config="$DOCKER_CONF" login -u="$RH_REGISTRY_USER" -p="$RH_REGISTRY_TOKEN" registry.redhat.io
 docker --config="$DOCKER_CONF" build -t "${IMAGE}:${IMAGE_TAG}" $APP_ROOT -f $APP_ROOT/Dockerfile
 docker --config="$DOCKER_CONF" push "${IMAGE}:${IMAGE_TAG}"
+docker --config="$DOCKER_CONF" tag "${IMAGE}:${IMAGE_TAG}" "${IMAGE}:${PR_LATEST}"
```

Comment:
You can pass multiple `-t` options to `docker build`, consider doing that so its just one push.

Reply:
So I tried.
In one push, it would be docker push IMAGE --all-tags. I don't think it should be done. It pushes the existing images as well. Changing the timestamp of older images as well :/

Reply:
Thanks for checking, good call in not pushing all tags.